src/main/java/org/gridsuite/loadflow/server/dto/LoadFlowStatus.java

```diff
@@ -0,0 +1,16 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.loadflow.server.dto;
+
+/**
+ * @author Anis Touri <anis.touri at rte-france.com
+ */
+public enum LoadFlowStatus {
+    NOT_DONE,
+    RUNNING,
+    COMPLETED
```

Comment:
You need to distinguish the CONVERGED and DIVERGED cases

Reply:
done